addons/GGS/src/components/keybind/keybind_confirm.gd

```diff
@@ -30,13 +30,24 @@ func _input(event: InputEvent) -> void:
 				return
 	
 	# Only continue when the key is being pressed
-	if not event.pressed:
+	if event.has_meta("pressed") and event.pressed == false:
```

Comment:
Is the `has_meta` really necessary here? I tried using `get_meta_list()` to print all of the meta info of the `event` but it just returns an empty array. Although, I only tested this with keyboard events. So I assume gamepad events (both button and motion) have meta info?

Reply:
I got errors for `pressed does not exist in event` when I first tried to run it, but I've also found that has_meta is inconsistent. It may not be the correct check here. In other places I opted to use `event is InputEventJoypadMotion`. I'll give that a shot instead

Reply:
Yeah. I guess we should check if the event is *not* an `InputEventJoypadMotion` before checking `event.pressed` because as far as I know, `pressed` does not exist on `InputEventJoypadMotion`. So maybe that's the issue.